feat: add the migration script to update safaricom_requests database table

ashishkumarbasak-accenture opened the pull request:
[AB#29906](https://dev.azure.com/redcrossnl/121%20Platform/_workitems/edit/29906)

## Describe your changes

Added a new migration script to drop un-necessary columns from `safaricom_requests` database table and keep only the required columns with renamed column names according to [this miro board](https://miro.com/app/board/uXjVKzM7CFE=/)

## Checklist before requesting a review

- [x] I have performed a self-review of my code
- [x] I have added tests wherever relevant
- [ ] I have made sure that all automated checks pass before requesting a review
- [x] I do not need any deviation from our PR guidelines

diderikvw commented on services/121-service/src/migration/1724754547895-modify-safaricom-requests-table.ts:
```diff
@@ -0,0 +1,66 @@
+    );
+    await queryRunner.query(
+      `ALTER TABLE "121-service"."safaricom_request" RENAME COLUMN "transactionId" TO "mpesaTransactionId"`,
+    );
```
Also rename column originatorConversationID to originatorConversationId (looks like a typo in the code and the diagram).

ashishkumarbasak-accenture replied:
Updated!

diderikvw reviewed (changes requested):
Migration script looks good: in principle it creates the data structure as it should.

Except:
1. A typo: see specific comment below.

2. When running the command to generate a migration, it still generates a migration script, which it should not:
![image](https://github.com/user-attachments/assets/a759ecf6-1c74-446d-93db-ad8f980aa840)

Please adjust the migration script so when running this command it comes up with nothing.

Note: it looks like generating a migration on main also results in a migration script: relating to the view and typeorm meta data. So that seems like a problem in main, not per-se this branch.

ashishkumarbasak-accenture commented:
> Migration script looks good: in principle it creates the data structure as it should.
> 
> Except:
> 
> 1. A typo: see specific comment below.
> 2. When running the command to generate a migration, it still generates a migration script, which it should not:
>    ![image](https://private-user-images.githubusercontent.com/1377493/362294203-a759ecf6-1c74-446d-93db-ad8f980aa840.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjUwMDExOTMsIm5iZiI6MTcyNTAwMDg5MywicGF0aCI6Ii8xMzc3NDkzLzM2MjI5NDIwMy1hNzU5ZWNmNi0xYzc0LTQ0NmQtOTNkYi1hZDhmOTgwYWE4NDAucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDgzMCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA4MzBUMDY1NDUzWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZDgyODYwNGI4NjA4ZWRjZTgzYjFmNWFmNDVkNTc2ZjE3NzFiMjkyNWU1YmM5ZGRjNGExNjAxOWJmYzAwNDQyNyZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.LIMVeIUCn8t3nDv7P1fiexQ87iwTOq2tS-e5q5PRlrc)
> 
> Please adjust the migration script so when running this command it comes up with nothing.
> 
> Note: it looks like generating a migration on main also results in a migration script: relating to the view and typeorm meta data. So that seems like a problem in main, not per-se this branch.

I have updated the MR.
Now the `npm run migration:generate` command is not generating migration script. I have fixed the issues related to my migration script & db table `safaricom_request`. Could you please recheck again.

![Screenshot 2024-08-30 at 08 56 00](https://github.com/user-attachments/assets/988f44a8-3e9e-40e7-8eb7-50cb7617f71a)
